jekyll/_cci2/style/formatting.adoc

```diff
@@ -4,19 +4,34 @@
 :toc: macro
 :toc-title:
 
+[#styling-text]
 == Styling text
 
 * Use **bold** for GUI menu options and button text. For example:
 ** Select **Organization Settings** from the sidebar
 ** Select **VCS** in the **Project Settings** window
 ** Select **Projects** from the sidebar and click **Set Up Project** for the project you want to follow
+* Also use **bold** to highlight key words in intro paragraphs. This helps with SEO.
```

Comment:
While this is true, I don't know how relevant it is to documentation. Do we do this anywhere? This can be confusing since we use bold to identify CirlceCI specific options as well. The question is, how useful to the user is bolding seemingly random words in intro paragraphs. Will they be more confused or more educated?

Reply:
Good question, this came from the web optimisation project I think. We don't do this ATM, it was a request for us to do so but we can totally discuss and decide it's not appropriate. Will continue in Slack!